Classes/ObjectExplorers/Views/FLEXTableView.m

```diff
@@ -18,8 +18,10 @@ @implementation FLEXTableView
 
 - (CGFloat)_heightForHeaderInSection:(NSInteger)section {
     CGFloat height = [super _heightForHeaderInSection:section];
-    if (section == 0 && self.tableHeaderView && !@available(iOS 13.0, *)) {
-        return height - self.tableHeaderView.frame.size.height + 8;
+    if (section == 0 && self.tableHeaderView) {
+        if (@available(iOS 13.0, *)) {} else {
+            return height - self.tableHeaderView.frame.size.height + 8;
+        }
```

Comment:
This is the sort of thing I wanted to avoid with `-Wno-unsupported-availability-guard`. Could you just add that flag to your project instead? I really don't want ugly workarounds like this just to avoid a (pointlessly pedantic) warning :/

Reply:
```
#pragma clang diagnostic push
#pragma clang diagnostic ignored "-Wunsupported-availability-guard"

#pragma clang diagnostic pop
```

^ What do you think if I add this? 
Developers can keep the flags in their projects as they want and we don't need to modify the if condition.

Reply:
I would really like to avoid needing all that extra code just to have some if statements look as they should. Personally, I think the warning is bogus and everyone should use this flag. I realize this is a huge OSS repo lots of people use though, so I want your feedback: is it asking too much for the minority of users who manually install FLEX into a project to add a flag to avoid this warning?

Reply:
ok, I will add it. Thanks for your time